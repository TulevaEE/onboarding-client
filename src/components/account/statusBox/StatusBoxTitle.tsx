import React from 'react';
import { FormattedMessage } from 'react-intl';

export const StatusBoxTitle: React.FunctionComponent = () => (
  <div className="row">
    <div className="col-md-6 mb-4 lead">
      <FormattedMessage id="account.status.choices" />
    </div>
  </div>
);
