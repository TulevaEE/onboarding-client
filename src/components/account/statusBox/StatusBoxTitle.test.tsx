import React from 'react';
import { shallow } from 'enzyme';
import { StatusBoxTitle } from './StatusBoxTitle';

describe('StatusBoxLoader', () => {
  const component = shallow(<StatusBoxTitle />);

  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
