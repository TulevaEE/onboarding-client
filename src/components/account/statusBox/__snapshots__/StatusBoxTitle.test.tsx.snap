// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBoxLoader renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-md-6 mb-4 lead"
  >
    <MemoizedFormattedMessage
      id="account.status.choices"
    />
  </div>
</div>
`;
